### PR TITLE
feat: [MSSQL] expose parameter_group_name

### PR DESCRIPTION
**Pull request by fnaranjo-vmw**
[#185201223](https://www.pivotaltracker.com/story/show/185201223)

Related PRs:
- https://github.com/cloudfoundry/csb-brokerpak-aws/pull/1110

Which other branches should this be merged with (if any)? None

### CODE REVIEW

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -81,6 +81,13 @@ New features and changes in this release:
   in the AWS documentation
   and [Configuration Parameters](reference/aws-mssql.html#parameters).
 
+  - **Parameter group:**
+  An existing database parameter group can now be associated through `parameter_group_name`.
+  If left unset, the default parameter group for the specified `mssql_version` is used.
```
Comment (zucchinidev) [inline release-notes.html.md.erb:86]: nit-pick: the parameter group used is what we created with the containment database configuration

We need to change the description

File: reference/aws-mssql.html.md.erb [hunk 2]
```diff
@@ -171,6 +171,19 @@ provision only:
       <td><code>None</code></td>
       <td>provision and update</td>
     </tr>
+    <tr>
+      <td><code>parameter_group_name</code></td>
+      <td>String</td>
+      <td>
+        Name of the custom parameter group to associate with this instance.
+        If left unset, the default parameter group for the specified mssql_version is used.
```
Comment (zucchinidev) [inline reference/aws-mssql.html.md.erb:179]: nit-pick: the parameter group used is what we created with the containment database configuration

We need to change the description

**Review by zucchinidev — CHANGES_REQUESTED** [review]

File: release-notes.html.md.erb (hunk 1 above)
Comment (fnaranjo-vmw) [reply release-notes.html.md.erb:86]: Great catch! Amending.
```suggestion
  If left unset, a new parameter group is created automatically with containment enabled.
```

File: reference/aws-mssql.html.md.erb (hunk 2 above)
Comment (fnaranjo-vmw) [reply reference/aws-mssql.html.md.erb:179]: Great catch! Amending.
```suggestion
        If left unset, a new parameter group is created automatically with containment enabled.
```

**Review by zucchinidev — APPROVED** [review]

**Pull request merged**